mizar/obj/endpoint.py

```diff
@@ -364,3 +375,109 @@ def update_agent_substrate(self, ep, bouncer):
 
     def delete_agent_substrate(self, ep, bouncer):
         self.rpc.delete_agent_substrate_ep(ep, bouncer.ip)
+
+    def update_networkpolicy_per_endpoint(self, data):
```

Comment:
Any reason you dont want include this in network policy?

Reply:
To calculate bitmap value for network policies, there is quantity limitation for policy. So we decided to limit data scope to be within an endpoint. There could be 1000 policies in a cluster, but we expect less than 100 policies for a certain endpoint. 
This function is to handle data for an endpoint. That's the reason putting the func here.